misc/DthTestProjects/EmptyConsoleApp/Program.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Misc.DthTestProjects.EmptyConsoleApp
+{
+    public class Program
+    {
+        public int Main(string[] args)
+        {
+            Console.WriteLine("Hail, Hydra.");
```

Comment:
Yeah, as much as I like amusing test code, it would probably be safer to go with something standard like, y'know "Hello, World"